### PR TITLE
Adds pre-set warp points

### PR DESCRIPTION
**Pull request by IamVelcroboy**
## About the PR
Title
Will work together with [this](https://github.com/DeltaV-Station/Delta-v/pull/968) to create a cleaner more straightforward WP/Beacon mapping system.

## Why / Balance
Easier for mappers. Better for players

### CODE REVIEW

**Review by Axionyxx — DISMISSED** [review]
Not a maintainer, but it looks good to me

File: Resources/Prototypes/DeltaV/Entities/Markers/warp_points.yml [hunk 1]
```diff
@@ -0,0 +1,338 @@
+# Off station
+- type: entity
+  id: WarpPointEvacShuttle
+  parent: WarpPoint
+  suffix: Evac Shuttle
+  components:
+  - type: WarpPoint
+    location: warp-point-evac
+  - type: Sprite
+    state: pink
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Markers/warp_points.yml]: why dupe this for every single warp point preset

Comment (IamVelcroboy) [reply Resources/Prototypes/DeltaV/Entities/Markers/warp_points.yml]: Good point

Comment (deltanedas) [reply Resources/Prototypes/DeltaV/Entities/Markers/warp_points.yml]: point like warp point?!?!?!?!

Comment (IamVelcroboy) [reply Resources/Prototypes/DeltaV/Entities/Markers/warp_points.yml]: Deltanedas makes great warp points!

**Pull request merged**